### PR TITLE
Add SmsPoh channel

### PR DESCRIPTION
**Pull request by tintnaingwinn**

### CODE REVIEW

**Comment by tintnaingwinn** [comment]
Hello @atymic, May I know how long will be take for review process?

File: src/SmspohChannel.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            return;
+        }
+
+        $message = $notification->toSmspoh($notifiable);
+
+        if (is_string($message)) {
+            $message = new SmspohMessage($message);
+        }
+
+        if (mb_strlen($message->content) > 160) {
```
Comment (atymic) [inline src/SmspohChannel.php]: In the docs is says

> This is the message body content. This parameter should be no longer than 6 message parts.

Generally most SMS providers can send messages longer than a single message, are you able to test this? If not, all good :)

**Comment by atymic** [comment]
Please PR your code against the new repo for final review :)
https://github.com/laravel-notification-channels/smspoh

File: src/SmspohApi.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        try {
+            $response = $this->client->request('POST', $this->endpoint, [
+                'headers' => [
+                    'Authorization' => "Bearer {$this->token}",
+                ],
+                'json' => [
+                    'sender' => Arr::get($message, 'sender'),
+                    'to' => Arr::get($message, 'to'),
+                    'message' => Arr::get($message, 'message'),
+                    'test' => Arr::get($message, 'test', 0),
```
[the remark this answers is not in the document]
Comment (tintnaingwinn) [reply src/SmspohApi.php]: It's boolean.
Screen Shot 2019-12-25 at 14 16 44

**Comment by tintnaingwinn** [comment]
Hi @atymic, I created PR at the new repo.

**Pull request closed without merging**